pysoa/common/serializer/msgpack_serializer.py

```diff
@@ -94,7 +94,7 @@ def dict_to_blob(self, data_dict):  # type: (Dict) -> six.binary_type
         if not isinstance(data_dict, dict):
             raise ValueError('Input must be a dict')
         try:
-            return msgpack.packb(data_dict, default=self._default, use_bin_type=True)
+            return msgpack.packb(data_dict, default=self._default, use_bin_type=True, unicode_errors='replace')
```

Comment:
we should use `surrogateescape` to tackle the specific error we are receiving. this library is used by ALL pysoa services so using `replace` might give us undesired effects elsewhere (where for instance we want the unicode encoding to fail)